src/schedules/dto/topic.dto.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class Topic {
+  @ApiProperty({
+    type: Number,
+  })
+  id: number;
+  @ApiProperty({
+    type: String,
+  })
+  name: string;
+  @ApiProperty({
+    type: String,
+  })
+  token: string;
+  @ApiProperty({
+    type: Date,
+  })
+  updated_at: Date;
+  @ApiProperty({
+    type: Date,
+  })
+  created_at: Date;
```

Comment:
Quebra linha entre as propriedades para deixar mais organizado, pf:

```suggestion
  @ApiProperty({
    type: Number,
  })
  id: number;
  
  @ApiProperty({
    type: String,
  })
  name: string;
  
  @ApiProperty({
    type: String,
  })
  token: string;
  
  @ApiProperty({
    type: Date,
  })
  updated_at: Date;
  
  @ApiProperty({
    type: Date,
  })
  created_at: Date;
```